common4j/build.gradle

```diff
@@ -219,6 +219,7 @@ dependencies {
     implementation "org.json:json:$rootProject.ext.jsonVersion"
     implementation "com.github.stephenc.jcip:jcip-annotations:$rootProject.ext.jcipAnnotationVersion"
     implementation "cz.msebera.android:httpclient:$rootProject.ext.mseberaApacheHttpClientVersion"
```

Comment:
so are we removing this in a separate PR?

Reply:
yep. There's part 2.